sensu-plugins-chef.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsChef::Version::VER_STRING
 
-  s.add_runtime_dependency 'chef', '12.12.15'
+  s.add_runtime_dependency 'chef', '= 14.8.12'
```

Comment:
I'm unhappy that Dependabot was inserting an equality sign here. Looks like the issue is specific to exact matchers in Gemspecs, and I've fixed it [here](https://github.com/dependabot/dependabot-core/commit/d33b3e39b78bdc7f221338d1f715ef73030da6e9).